### ISSUE TITLE
What are the possible reasons for the big gap between the use of LBFGS and Adam optimizer?（什么可能的原因造成使用LBFGS和Adam优化器差距很大？）

### ISSUE THREAD

**Issue by glotm** [KindXiaoming/pykan#459]
# Data
![image](https://github.com/user-attachments/assets/5ec087a7-0708-41f3-889d-eb416b4c4549)

The function is ,

$$
y=exp(0.1x_1+0.12x_2+exp(0.2x_3+0.5x_4))
$$ 

# KAN define
![image](https://github.com/user-attachments/assets/aec59070-b3b5-465c-a84c-283ebf4196b7)

![image](https://github.com/user-attachments/assets/fd13c34d-bd1f-4fd7-ba91-dcd0b0d6803e)
![image](https://github.com/user-attachments/assets/f14ccaba-dc48-437a-8384-57786490bdf2)

When I continue to train, there is still a big gap in the subsequent losses. I would like to know why?

**Comment by vrbaj** [comment]
LBFGS is second order method which usually converges faster than Adam. Probably tweaking Adam lr would improve its performance. LBFGS does not use learning rate btw.

**Comment by kunliu916** [comment]
I have encountered the same problem, too.Dose anyone know how to enable the fused Pytorch learning rate scheduler?
